### PR TITLE
Final fixes for DSChief

### PR DESCRIPTION
**Pull request by norhh**

### CODE REVIEW

File: mythril/laser/ethereum/keccak_function_manager.py [hunk 1]
```diff
@@ -33,6 +33,7 @@ def __init__(self):
         self.store_function = {}  # type: Dict[int, Tuple[Function, Function]]
         self.interval_hook_for_size = {}  # type: Dict[int, int]
         self._index_counter = TOTAL_PARTS - 34534
+        self.store_hashes = {}  # type: Dict[int, List[BitVec]]
```
Comment (JoranHonig) [inline mythril/laser/ethereum/keccak_function_manager.py]: What about `hash_result_store`. (this is fairly subjective so I don't mind if you don't do anything with this comment)
This seems slightly more clear on the purpose of the variable

File: mythril/analysis/solver.py [hunk 2]
```diff
@@ -138,35 +138,28 @@ def get_transaction_sequence(
...
+    concrete_hashes = keccak_function_manager.get_concrete_hash_data(model)
     for tx in concrete_transactions:
         if hash_matcher not in tx["input"]:
             continue
         if code is not None and code.bytecode in tx["input"]:
             s_index = len(code.bytecode) + 2
         else:
             s_index = 10
-        for i in range(s_index, len(tx["input"])):
+        for i in range(s_index, len(tx["input"]), 64):
```
Comment (JoranHonig) [inline mythril/analysis/solver.py]: Why did you add back the alignment again?
(with the current abi spec this won't be a problem, but I wonder 😅 )

File: mythril/laser/ethereum/keccak_function_manager.py [hunk 3]
```diff
@@ -85,8 +87,27 @@ def create_keccak(self, data: BitVec) -> Tuple[BitVec, Bool]:
...
+        concrete_hashes = {}  # type: Dict[int, List[Optional[int]]]
+        for size in self.store_hashes:
+            concrete_hashes[size] = []
+            for val in self.store_hashes[size]:
+                eval = model.eval(val.raw)
+                try:
+                    concrete_val = eval.as_long()
+                except AttributeError:
+                    concrete_val = None
+                concrete_hashes[size].append(concrete_val)
```
[the remark this answers is not in the document]
Comment (norhh) [reply mythril/laser/ethereum/keccak_function_manager.py]: Yeah nevermind, this is what happens when coding at night.

File: mythril/analysis/solver.py (hunk 2 above)
Comment (norhh) [reply mythril/analysis/solver.py]: This probably popped up while I was checking where the problem was, will remove this line

File: mythril/laser/ethereum/keccak_function_manager.py (hunk 1 above)
Comment (norhh) [reply mythril/laser/ethereum/keccak_function_manager.py]: I am in class now. Changing this on mobile phone is gonna be tricky, feel free to commit it to this pr( or I can do it few hrs later if there is no urgency)

File: mythril/laser/ethereum/keccak_function_manager.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (JoranHonig) [reply mythril/laser/ethereum/keccak_function_manager.py]: I submitted a commit to fix this
